recipes/pweave/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "0.25" %}
+{% set sha256 = "1c0f6921196646243eb7ff9eee742305909be2bc7a5eeeb06a7d1f66cc9758c7" %}
+
+package:
+  name: pweave
+  version: {{ version }}
+
+source:
+  fn: Pweave-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pweave/Pweave-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - Pweave = pweave.scripts:weave
+    - pweave = pweave.scripts:weave
+    - Ptangle = pweave.scripts:tangle
+    - ptangle = pweave.scripts:tangle
+    - pypublish = pweave.scripts:publish
+    - pweave-convert = pweave.scripts:convert
```

Comment:
We should switch to using pip to install once conda 4.2.3 or above is stable.  Pip handles entry points much better, and then we really could just drop the entry points from meta.yaml.